community-containers/jellyfin/readme.md

```diff
@@ -0,0 +1,14 @@
+## Jellyfin
+This container bundles Jellyfin and auto-configures it for you.
+
+### Notes
+- This is not working on Docker Desktop since it needs `network_mode: host` in order to work correctly.
```

Comment:
We also need some advices that this container is incopatible with Plex community container and that you need to have your own reverse proxy or use the caddy community container. Also need to adjust the readme of the caddy community container and plex community container then...